app/css/components/code-pane.css

```diff
@@ -5,7 +5,7 @@
         @apply flex;
         @apply py-8;
         ul {
-            @apply w-fill;
+            width: fit-content;
```

Comment:
I think this is wrong. It means the whole line doesn't get highlighted on hover - just the first bit of it that has text.

<img width="917" alt="Screenshot 2023-06-01 at 13 29 58" src="https://github.com/exercism/website/assets/286476/3af9ad32-20cc-4b90-af85-0068a3de903f">